go-controller/pkg/ovn/ovn.go

```diff
@@ -141,6 +138,9 @@ type Controller struct {
 	namespaces      map[string]*namespaceInfo
 	namespacesMutex sync.Mutex
 
+	egressGWCache      map[ktypes.NamespacedName]*externalRouteInfo
```

Comment:
should probably rename this to exGWCache

Reply:
yes please, we have too many `egress` terminologies for IPs, firewalls, routers..

Reply:
done